app/containers/markdown/Link.js

```diff
@@ -18,10 +21,16 @@ const Link = React.memo(({
 
 	const childLength = React.Children.toArray(children).filter(o => o).length;
 
+	const onLongPress = () => {
+		Clipboard.setString(link);
+		EventEmitter.emit(LISTENER, { message: I18n.t('Copied_to_clipboard') });
+	};
+
 	// if you have a [](https://rocket.chat) render https://rocket.chat
 	return (
 		<Text
 			onPress={preview ? undefined : handlePress}
+			onLongPress={onLongPress}
```

Comment:
You need to handle preview cases.

Reply:
Preview Cases are handled by the onPress function already.

Reply:
No, we not want that when user long press a link on roomsListView this link are copy. Use some condition like on `onPress`.

Reply:
Ok, I will apply some conditions